proof/crefine/RISCV64/IpcCancel_C.thy

```diff
@@ -2815,13 +2935,14 @@ sorry (* FIXME RT: reply_remove_ccorres *)
 lemma cancelIPC_ccorres1:
   assumes cteDeleteOne_ccorres:
   "\<And>w slot. ccorres dc xfdc
-   (invs' and cte_wp_at' (\<lambda>ct. w = -1 \<or> cteCap ct = NullCap
+   (invs'
+    and cte_wp_at' (\<lambda>ct. w = -1 \<or> cteCap ct = NullCap
         \<or> (\<forall>cap'. ccap_relation (cteCap ct) cap' \<longrightarrow> cap_get_tag cap' = w)) slot)
```

Comment:
This indentation doesn't seem quite right yet, at first glance I was still confused before realising that the final `∨` is inside the `cte_wp_at'`. This looks like a bit of a painful one to do properly without some awkward linebreaks, but at the very least I think the final line should be indented two more.

Reply:
Ah I misread this bit. I can open it up and redo it